coral/src/app/features/topics/details/overview/components/TopicPromotionBanner.tsx

```diff
@@ -1,98 +1,39 @@
+   * it can't be promoted, and we want to show that
+   * information to the user.
+   */
+  hasOpenTopicRequest: boolean;
+  topicName: string;
 }
 
 const TopicPromotionBanner = ({
-  isTopicOwner,
   topicPromotionDetails,
-  hasOpenRequest,
+  hasOpenTopicRequest,
+  topicName,
 }: TopicPromotionBannerProps) => {
-  const navigate = useNavigate();
-  const { status, targetEnv, sourceEnv, targetEnvId, topicName } =
-    topicPromotionDetails;
-
-  const showRequestPromotionBanner =
-    isTopicOwner &&
-    status === "SUCCESS" &&
-    targetEnv !== undefined &&
-    sourceEnv !== undefined &&
-    targetEnvId !== undefined;
-
-  // hasOpenTopicRequest is true for any request open on the current topic in the current environment (source environment)
-  if (hasOpenRequest) {
-    return (
-      <GridItem colSpan={"span-2"}>
-        <Banner image={illustration} layout="vertical" title={""}>
-          <Box component={"p"} marginBottom={"l1"}>
-            There is an open request for {topicName}.
-          </Box>
-          <Button.Primary
-            onClick={() =>
-              navigate(
-                `/requests/topics?search=${topicName}&status=CREATED&page=1`
-              )
-            }
-          >
-            See the request
-          </Button.Primary>
-        </Banner>
-      </GridItem>
-    );
-  }
-
-  // status is "REQUEST_OPEN" when there is a promotion request open on a topic in the target environment for promotion,
-  // ie not the current topic + source environment
-  if (status === "REQUEST_OPEN") {
-    return (
-      <GridItem colSpan={"span-2"}>
-        <Banner image={illustration} layout="vertical" title={""}>
-          <Box component={"p"} marginBottom={"l1"}>
-            There is already an open promotion request for {topicName}.
-          </Box>
-          <Button.Primary
-            onClick={() =>
-              navigate(
-                `/requests/topics?search=${topicName}&requestType=PROMOTE&status=CREATED&page=1`
-              )
-            }
-          >
-            See the request
-          </Button.Primary>
-        </Banner>
-      </GridItem>
-    );
-  }
-
-  if (showRequestPromotionBanner) {
-    return (
-      <GridItem colSpan={"span-2"}>
-        <Banner image={illustration} layout="vertical" title={""}>
-          <Box component={"p"} marginBottom={"l1"}>
-            This schema has not yet been promoted to the {targetEnv}{" "}
-            environment.
-          </Box>
-          <Button.Primary
-            onClick={() =>
-              navigate(
-                `/topic/${topicName}/request-promotion?sourceEnv=${sourceEnv}&targetEnv=${targetEnvId}`
-              )
-            }
+  return (
+    <div data-testid={"topic-promotion-banner"}>
+      <PromotionBanner
+        topicName={topicName}
+        promotionDetails={topicPromotionDetails}
+        hasOpenRequest={hasOpenTopicRequest}
+        type={"topic"}
+        promoteElement={
+          <Button.ExternalLink
```

Comment:
I believe `Button.ExternalLink` does not work when Klaw is built for deployment. The URL will not add the `coral` prefix correctly. That's the reason for the navigate on click pattern, which is incorrect, but not broken :/

Reply:
DANG I vaguely remembered something about that, but thought it was somehow related to the feature flags 🤦  Will check!

Reply:
HA! Added a PR: https://github.com/aiven/klaw/pull/1520